app/views/welcome/index.html.haml

```diff
@@ -0,0 +1,3 @@
+= render "./partials/navbar"
```

Comment:
Maybe this should be rendered in the listings view going forward, meaning we do not need a welcome/index view for rendering it?

Reply:
listings controller created

Reply:
This should be rendered on the application layout. Not in ANY view.